lua/toggleterm.lua

```diff
@@ -189,10 +189,10 @@ end
 --- @param terminal_id number
 function M.send_lines_to_terminal(selection_type, trim_spaces, terminal_id)
   -- trim_spaces defines if we should trim the spaces from lines which are sent to the terminal
-  if trim_spaces == nil then trim_spaces = true end
+  trim_spaces = trim_spaces or true
```

Comment:
this will need an explicit `nil` check since what can happen here is `trim_space` can be `false` and it will then trigger the `or` clause whereas if you do an explicit `nil` check it will work the way you intended
i.e.
```lua
trim_spaces = trim_spaces == nil or trim_spaces
```

Reply:
Oh my... thanks for catching it. fixed